src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/Controllers/Controller.tsx

```diff
@@ -10,8 +10,7 @@ import { UpgradeFormFields } from '../UpgradeForm'
 
 interface BillingControlsProps {
   seats: number
-  newPlan?: PlanName
-  selectedPlan: PlanName
+  newPlan?: Plan
```

Comment:
This now takes a plan, and selectedPlan is no longer needed in this function either.

IN CASE ANYONE STUMBLES UPON THIS COMMENT LATER:
- SelectedPlan never "changes values" like newPlan does... so having newPlan be the source of truth here makes the most sense